examples/rust_pubsub/src/simple_publisher.rs

```diff
@@ -0,0 +1,38 @@
+use rclrs::{create_node, Context, Node, Publisher, RclrsError, QOS_PROFILE_DEFAULT};
+use std::{sync::Arc, thread, time::Duration};
+use std_msgs::msg::String as StringMsg;
+// / SimplePublisherNode struct contains node and publisher members.
+// / Used to initialize a ROS 2 node and publisher, and publish messages.
```

Comment:
Is this intended to be documentation? If so, you should move the 3rd slash over, like this `///`

Also, I think  this documentation in the example code itself could come at a later time. Its a little odd that this is the only documentation in the example itself. 

```suggestion
```